install.rdf

```diff
@@ -6,7 +6,7 @@
     <em:hasEmbeddedWebExtension>true</em:hasEmbeddedWebExtension>
     <em:type>2</em:type>
     <em:unpack>false</em:unpack>
-    <em:version>1.0.10</em:version>
+    <em:version>0.4.0</em:version>
```

Comment:
I am a little worried about this tbh, this pr was just to get them in sync. I will need to do a version bump for a release later this week (or early next week) with the changes from https://github.com/meandavejustice/min-vid/milestone/10. We should figure this out for certain before then.